jormungandr/src/settings/command_arguments.rs

```diff
@@ -99,3 +104,7 @@ impl CommandLine {
         Self::from_args()
     }
 }
+
+fn log_level_parse(level: &str) -> Result<FilterLevel, &'static str> {
+    level.parse().map_err(|_| "")
```

Comment:
Hmm... nothing to output here? Could it be `.map_err(|e| format!("{}", e))`?

Reply:
The output doesn't matter here, it's ignored by Clap anyway. It's automatically generating a nice error message based on list of possible variants. It's mapped from `()` to `&str` only to fulfill the `Display` requirement.

Reply:
Strange that it would require `Display` and not use it. I'd leave a comment explaining why the message does not matter.